### PR TITLE
Add support for LV95

### PR DESCRIPTION
**Pull request by loicgasser**
[Test Link Get Feature 2056](https://mf-chsdi3.dev.bgdi.ch/gal_lv95/rest/services/ech/MapServer/ch.bafu.bundesinventare-bln/1?sr=2056)
[Test Link Identify 2056](https://mf-chsdi3.dev.bgdi.ch/gal_lv95/rest/services/ech/MapServer/identify?geometryType=esriGeometryEnvelope&geometry=2545000,1145000,2555000,1155000&imageDisplay=500,600,96&mapExtent=548945.5,147956,549402,148103.5&tolerance=1&layers=all:ch.bafu.bundesinventare-bln&sr=2056)

Works with https://github.com/geoadmin/service-sphinxsearch/pull/331

### CODE REVIEW

**Comment by loicgasser** [comment]
@procrastinatio I'd be interested if you could give me your input on the approach.

File: chsdi/lib/validation/identify.py [hunk 1]
```diff
@@ -148,8 +153,7 @@ def mapExtent(self, value):
             raise HTTPBadRequest('Please provide the parameter mapExtent  (Required)')
         else:
             try:
-                feat = loads(value)
-                self._mapExtent = asShape(feat)
+                self._mapExtent = esrijson.to_shape([float_raise_nan(c) for c in value.split(',')])
             except ValueError:
```
Comment (loicgasser) [inline chsdi/lib/validation/identify.py]: use generic exception instead

**Comment by loicgasser** [comment]
@procrastinatio @gjn please review I am done with it.

**Comment by loicgasser** [comment]
And I don't think we need to wait for TOD, this is just the support for LV95.

**Comment by gjn** [comment]
https://github.com/geoadmin/service-sphinxsearch/pull/331 is needed to be deployed first (re-create all sphinx indices, then we can do a silent release of this PR.

**Comment by loicgasser** [comment]
Fix for https://github.com/geoadmin/mf-chsdi3/issues/1336
Fix for https://github.com/geoadmin/mf-chsdi3/issues/1965

**Comment by gjn** [comment]
I've checkt out this branch under my account. I have something strange. The map in the extended tooltip is working on the deployed branch:
https://mf-chsdi3.dev.bgdi.ch/gal_lv95/rest/services/ech/MapServer/ch.bav.haltestellen-oev/8591514/extendedHtmlPopup?lang=en

But it doesn't work on my env:
https://mf-chsdi3.dev.bgdi.ch/ltjeg/rest/services/ech/MapServer/ch.bav.haltestellen-oev/8591514/extendedHtmlPopup?lang=en

Any idea why?

**Comment by loicgasser** [comment]
I messed up the rebase give me a minute

**Comment by loicgasser** [comment]
it's related to your geoadmin env
//mf-geoadmin3.dev.bgdi.ch/ltjeg/embed.html?ch.bav.haltestellen-oev=8591514&lang=en&topic=ech

vs

//mf-geoadmin3.dev.bgdi.ch/embed.html?ch.bav.haltestellen-oev=8591514&lang=en&topic=ech

**Comment by gjn** [comment]
ok. Thanks.

**Comment by ltclm** [comment]
jenkins test

**Comment by loicgasser** [comment]
This was merged a long time ago. :D

**Pull request merged**